windows builds failing during autobuild

Autobuild link: http://charm.cs.illinois.edu/autobuild/old.2020_01_13__01_02/mpi-win-x86_64.txt

This is possibly because of a build failure while compiling charmrun.

```
make[1]: Leaving directory '/home/nikhil/autobuild/netlrts-win-x86_64-smp/charm/netlrts-win-x86_64-smp/tmp/charmrun-src'
make[1]: Target 'all' not remade because of errors.
make: *** [Makefile.machine:2: charmrun] Error 2
```

--- a/src/arch/mpi-win64/charmrun
+++ b/src/arch/mpi-win64/charmrun
@@ -79,7 +79,7 @@ do
 	++debugger)
 		DEBUGGER=$2
 		shift
-        ;;
+		;;
 	*) 
 	    args=$args" "$1
 	    ;;

--- a/src/arch/mpi-win64/charmrun
+++ b/src/arch/mpi-win64/charmrun
@@ -79,6 +79,7 @@ do
 	++debugger)
 		DEBUGGER=$2
 		shift
+        ;;
 	*) 
 	    args=$args" "$1
 	    ;;

